src/core/public/public.api.md

```diff
@@ -205,87 +199,25 @@ export interface I18nSetup {
 export type I18nStart = I18nSetup;
 
 // @internal (undocumented)
-export interface InjectedMetadataParams {
-    // (undocumented)
-    injectedMetadata: {
-        version: string;
-        buildNumber: number;
-        basePath: string;
-        csp: {
-            warnLegacyBrowsers: boolean;
-        };
-        vars: {
-            [key: string]: unknown;
-        };
-        uiPlugins: Array<{
-            id: PluginName;
-            plugin: DiscoveredPlugin;
-        }>;
-        legacyMetadata: {
-            app: unknown;
-            translations: unknown;
-            bundleId: string;
-            nav: LegacyNavLink[];
-            version: string;
-            branch: string;
-            buildNum: number;
-            buildSha: string;
-            basePath: string;
-            serverName: string;
-            devMode: boolean;
-            uiSettings: {
-                defaults: UiSettingsState;
-                user?: UiSettingsState;
-            };
-        };
-    };
-}
-
-// @public
-export interface InjectedMetadataSetup {
-    // (undocumented)
-    getBasePath: () => string;
+export interface InternalCoreSetup extends CoreSetup {
     // (undocumented)
-    getCspConfig: () => {
-        warnLegacyBrowsers: boolean;
-    };
+    application: ApplicationSetup;
+    // Warning: (ae-forgotten-export) The symbol "InjectedMetadataSetup" needs to be exported by the entry point index.d.ts

```

Comment:
I see some of these warnings, are they anything to be worried about?

Reply:
This means `InjectedMetadataSetup` isn't being exported by the index. Since this PR moves it to `InternalCoreSetup` other plugins won't need to import the type from `src/core/public/index.ts` so we can safely ignore this.